src/components/Analytics/gtag.ts

```diff
@@ -0,0 +1,15 @@
+import { EventCategory } from './utils';
+
+// https://developers.google.com/analytics/devguides/collection/gtagjs/events
+export const event = (
+  action: string,
+  category: EventCategory,
+  label?: string,
+  value?: number,
+) => {
+  gtag('event', action, {
+    event_category: category,
+    event_label: label,
+    event_value: value,
+  });
+};
```

Comment:
A few notes here:

Let's make `event_label` required to ensure that we know what we are recording. Also, maybe rename the function name to `trackEvent` or `trackGA4Event` so the name better represents what the function does.

Finally, it's possible (but unlikely) that `gtag` when we call this function, consider adding a conditional to cover this case

```ts
if (gtag) {
  gtag('event', action, {/* ... */});
}
```